src/app/components/cards/BannerCard.tsx

```diff
@@ -0,0 +1,64 @@
+"use client";
+
+// components
+import Image from "next/image";
+import { Card, CardBody, CardHeader, CardMedia } from "@trussworks/react-uswds";
+import { CardIcon } from "./CardIcon";
+// utils
+import {
+  maskCurrency,
+  maskThousands,
+  maskPercentage,
+} from "../../utils/masking";
+//types
+import { College } from "../../types";
+
+export const BannerCard = ({ college }: Props) => {
+  return (
+    <Card layout="flagDefault" headerFirst={true} className="card card_banner">
+      <div className="card_banner-container">
+        <CardMedia>
+          <Image
+            src={`https://swift-institution-images-public.s3.amazonaws.com/${college.id}.png`}
+            alt={`AI generated image of ${college.name}`}
+            width={400}
+            height={400}
+          />
+        </CardMedia>
+        <CardBody>
+          <CardHeader className="card_header">
+            <h2 className="card_header-title">{college.name}</h2>
+            <p className="card_header-subtitle">{`${college.city}, ${college.state}`}</p>
+          </CardHeader>
+          <p className="card_desc">{college.description}</p>
+          <div className="card_grid">
+            <CardIcon
+              subtitle={"Type"}
+              highlight={college.type}
+              icon="account_balance"
+            />
+            <CardIcon
+              subtitle={"Undergraduate population"}
+              highlight={maskThousands(college.population)}
+              icon="people"
+            />
+            <CardIcon
+              subtitle={"Graduation rate"}
+              highlight={maskPercentage(college.completionRate)}
+              icon="school"
+            />
+            <CardIcon
+              subtitle={"Average cost per year"}
+              highlight={maskCurrency(college.avgCost)}
+              icon="local_offer"
+            />
+          </div>
+        </CardBody>
+      </div>
+    </Card>
+  );
+};
```

Comment:
do we still want to have an apply button from this page?

Reply:
Gooooood catch. Added that back in!